com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ExtenderUtil.java

```diff
@@ -797,6 +803,13 @@ public static void storeResources(File targetDirectory, Map<String, IResource> r
         }
     }
 
+    private static boolean isBinZip(String resourcePath) {
+        return resourcePath.endsWith("plugins/bin/common.zip") || Stream
+                .of(Platform.X86_64MacOS, Platform.X86_64Linux, Platform.X86_64Win32)
+                .flatMap(platform -> Stream.of(platform.getPair(), platform.getOs()))
+                .anyMatch(platformOption -> resourcePath.endsWith("plugins/bin/" + platformOption + ".zip"));
```

Comment:
Previously we've stuck with the platform name as the folder.
But I guess it makes sense to put them in the same folder then since they  themselves contain folder structures.

However, since they now are "generic", I'm not sure we should restrict us to the "bin" folder?
WHy not put them in the "plugins" folder, and then the zip file can contain files from any level in the folder structure.
E.g. what if the plugin contains .dll or .dylib that should go into the lib/ folder?

Reply:
The problem binzips are solving is that some binary executables require a lot of lua files alongside them on a disc to work. This messes with the editor and potentially makes bob compile and bundle those lua files to the game. `.dll` and `.dylib` files, however, don't mess with the editor, and as you said earlier, it's easier to work with files than with archives, so those can be stored in a lib folder as is, without being zipped.

With that said, do you think it still makes sense to make these archives more generic?

Btw, what do you mean by platform name? Is it only e.g. `macos` or `linux`, i.e. os name?